examples/gallery/histograms/blockm.py

```diff
@@ -0,0 +1,63 @@
+"""
+Blockmean
+---------
+The :meth:`pygmt.blockmean` method allows to calculate different quantities
```

Comment:
```suggestion
The :meth:`pygmt.blockmean` method calculates different quantities
```
The grammar is off with this sentence. Here is my suggested fix.